src/sentry/api/endpoints/organization_discover.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import absolute_import
+
+from datetime import datetime
+
+from rest_framework.response import Response
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationDiscoverPermission
+from sentry.models import Project, ProjectStatus
+
+from sentry.utils import snuba
+
+from django.core.exceptions import ValidationError
+
+
+class OrganizationDiscoverEndpoint(OrganizationEndpoint):
+    permission_classes = (OrganizationDiscoverPermission, )
+
+    def do_query(self, start, end, selected_columns=[], conditions=[], having=[], filters={},
+                 aggregations=[], orderby='-last_seen', limit=None, groupby=[], rollup=None):
```

Comment:
you don't want to have mutable data structures as the default args in python, see: http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments